### PR TITLE
Larissa edits

### LINKED ISSUE #42: Larissa edits
[Edits made by Larissa](https://docs.google.com/document/d/1BrKqZlTe_xmUJzp0Lt-vRGPAwIhU9obWTVwojRaRFBM/edit?ts=5722693c)

### PR DESCRIPTION
**Pull request by julienchastang**
This closes #42.

### CODE REVIEW

File: choosing.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ applications of environments:
 - A disposable environment for testing a new library without polluting the rest
   of the Python install; this environment can easily be discarded when no
   longer needed.
-- Quick set up of a known set of dependencies for workshops and tutorials
+- Quick set up of a known set of dependencies for workshops and tutorials.
```
Comment (dopplershift) [inline choosing.md]: This is not currently a sentence, so no period. That also goes for the one above.

**Comment by dopplershift** [comment]
Did you accidentally base this off your other PR?

File: choosing.md (hunk 1 above)
Comment (julienchastang) [inline choosing.md]: ok

**Comment by julienchastang** [comment]
Not really, but rebased off of origin/gh-pages anyway.

**Pull request merged**